### ISSUE TITLE
Enhancement Request: Provide fee limit assistance

### ISSUE THREAD

**Issue by dekdynamics** [ZeusLN/zeus#920]
**Describe the feature**

When paying a lightning invoice from Zeus, the user has a box to input "Fee limit (sats)." While this is useful, it could be difficult to calculate an appropriate fee while trying to go through a checkout aisle. Entering too low of a fee could result in long wait times, or more likely, payment failure. 

This enhancement request is to add three or four small square buttons directly below the "Fee limit (sats)" input field. The buttons would read something like...

```
0.01%       0.05%       0.1%      0.25%
```

Upon pressing a button, the "Fee limit (sats)" input box would be filled with the fee calculated by `$invoice_amount * $fee_rate`

From here the user can decide to manually adjust it, if needed.

I just loaded a fresh invoice for 1,000,000 sats, and the auto-filled fee is 10 sats. The likelihood of this payment succeeding is near zero. I think a quick calculation would help the user pick something more appropriate.

-------

Coming soon to LND is a new [time preference parameter](https://github.com/lightningnetwork/lnd/pull/6024), which would allow the user to select an option for their payment:

> -1: Optimize for fees only
> 0: Default trade-off based on the attempt cost configuration settings (attemptcost and attemptcostppm)
> 1: Optimize for reliability only (higher reliability is likely to reduce the time it takes to complete the payment).

Perhaps something like that is worth consideration as well.

**Comment by dekdynamics** [comment]
I had an estimated fee issue bite me recently, where I think this feature request could help. The recommended fee in Zeus showed at 2 sats, but I could not get a payment through no matter what I tried. Zues is connected to a private node with a single connection to The Wall - which is very well connected.

On the command line of my private node, I checked the possible routes with two commands:

`lncli queryroutes --use_mc --time_pref=-1 <pubkey> <amount>`
`lncli queryroutes --use_mc --time_pref=1 <pubkey> <amount>`

The low fee option showed me the same fee as Zeus estimated, but the success probability was under 0.2. The optimize for reliability option showed me a fee of 40 sats and a probability of 0.6. The fee ended up being 32 sats, which worked out to 680ppm. Totally reasonable, in my opinion.
